app/src/main/res/values/strings.xml

```diff
@@ -35,5 +35,22 @@
     <string name="scenario1_explanation">Verbind het lampje met de batterij, zodat het lampje aan gaat.</string>
     <string name="scenario2_explanation">Voeg een weerstand toe aan het circuit, zodat je het lampje niet kapot gaat. Om een weerstand toe te voegen, moet je een draad weg halen. Dit kan je doen door in draad modus op een draad te tikken. Voer het circuit uit als je klaar bent.</string>
     <string name="scenario_complete">Goed gedaan!</string>
+    <string name="scenario_not_complete">Je bent nog niet klaar!</string>
```

Comment:
Ja ik denk het wel. We zouden het natuurlijk ook zo kunnen doen dat we andere feedback geven op het moment dat de gebruiker nog niet alle componenten heeft geplaatst. Zou opzich niet zo moeilijk te controleren moeten zijn.